Tor proxy that can be deployed into Akash

jatm80 opened the pull request:
I tested this proxy in Akash network and works great  

[Proxy available at ](http://provider.akash.virtualhosting.hk:30753)

andy108369 commented:
@jatm80 thank you for your PR!
Please update the root level README.md before we can merge it.

jatm80 commented:
> @jatm80 thank you for your PR! Please update the root level README.md before we can merge it.

No problem, I just updated it.

andy108369 commented:
> > @jatm80 thank you for your PR! Please update the root level README.md before we can merge it.
> 
> No problem, I just updated it.

@jatm80  Looks great! Please add the last bit - `signedBy` attributes, e.g. https://github.com/akash-network/awesome-akash/blob/b720cd5/MySQL/deploy.yaml#L29-L32 to your deployment SDL manifest.

Thank you!

jatm80 commented:
> > > @jatm80 thank you for your PR! Please update the root level README.md before we can merge it.
> > 
> > 
> > No problem, I just updated it.
> 
> @jatm80 Looks great! Please add the last bit - `signedBy` attributes, e.g. https://github.com/akash-network/awesome-akash/blob/b720cd5/MySQL/deploy.yaml#L29-L32 to your deployment SDL manifest.
> 
> Thank you!

Sounds good,  I just added signedBy and attributes following the example.

Dimokus88 commented:
Ready to merge @andy108369
